app/views/devices/index.html.erb

```diff
@@ -0,0 +1,84 @@
+<%= render 'shared/partials/nav-global.html' %>
```

Comment:
I added some layout changes in the PR: `Added Layout Specific for Landing Page`.

As a quick side note, partials should be rendered in the following format:
<%= render partial: "shared/partials/nav-global" %>